CryptKeeperZK/crypt-keeper-extension
# feat: disable random identity

## Explanation

This PR hides under the feature flag random identity creation.

Details are below:
- [x] Add feature flags
- [x] Hide identity type select when creating new identity
- [x] Hide created random identities
- [x] Set env variables and setup test fixes

## More Information

Related to #184

## Screenshots/Screencaps

### Before

![image](https://user-images.githubusercontent.com/14254374/229585800-7319cb32-5218-4185-9976-2a91f0e0b567.png)
![image](https://user-images.githubusercontent.com/14254374/229585935-09a21638-7e40-47c7-9108-674952c25805.png)


### After

![image](https://user-images.githubusercontent.com/14254374/229585800-7319cb32-5218-4185-9976-2a91f0e0b567.png)
![image](https://user-images.githubusercontent.com/14254374/229585568-6d60ddd0-51b8-4249-861c-b17a3ae03184.png)


## Manual Testing Steps

1. Disable random identity feature like with env variable in `.env` file (see `.env.example`).
2. Check the create identity screen and home screen
3. Enabled random identity feature and check step 1 and 2. 

## Pre-Merge Checklist


### + If there are functional changes:


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
